packages/shared/src/utils/constants/contract.tsx

```diff
@@ -399,17 +400,17 @@ export const getSupportedContracts = is_high_low => ({
     },
 });
 
-export const getContractConfig = is_high_low => ({
+export const getContractConfig = (is_high_low: boolean) => ({
     ...getSupportedContracts(is_high_low),
     ...getUnsupportedContracts(),
 });
 
-export const getContractTypeDisplay = (type, is_high_low = false) => {
-    return getContractConfig(is_high_low)[type] ? getContractConfig(is_high_low)[type.toUpperCase()].name : '';
+export const getContractTypeDisplay = (type: TGetSupportedContracts, is_high_low = false) => {
+    return getContractConfig(is_high_low)[type].name || '';
 };
 
-export const getContractTypePosition = (type, is_high_low = false) =>
-    getContractConfig(is_high_low)[type] ? getContractConfig(is_high_low)[type.toUpperCase()].position : 'top';
+export const getContractTypePosition = (type: TGetSupportedContracts, is_high_low = false) =>
+    getContractConfig(is_high_low)[type].position || 'top';
 
-export const isCallPut = trade_type =>
+export const isCallPut = (trade_type: 'rise_fall' | 'rise_fall_equal' | 'high_low') =>
```

Comment:
check please  here and in other similar places if we should point return types in function.

export const isCallPut = (trade_type: 'rise_fall' | 'rise_fall_equal' | 'high_low'): boolean =>
trade_type === 'rise_fall' || trade_type === 'rise_fall_equal' || trade_type === 'high_low';